[2021-02-23 14:57:08,048] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_0 2021-02-22T00:00:00+00:00 [queued]>
[2021-02-23 14:57:08,915] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: example_bash_operator.runme_0 2021-02-22T00:00:00+00:00 [queued]>
[2021-02-23 14:57:08,918] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-02-23 14:57:08,927] {taskinstance.py:1043} INFO - Starting attempt 1 of 1
[2021-02-23 14:57:08,934] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-02-23 14:57:09,531] {taskinstance.py:1063} INFO - Executing <Task(BashOperator): runme_0> on 2021-02-22T00:00:00+00:00
[2021-02-23 14:57:09,741] {standard_task_runner.py:52} INFO - Started process 99 to run task
[2021-02-23 14:57:10,238] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'example_bash_operator', 'runme_0', '2021-02-22T00:00:00+00:00', '--job-id', '23', '--pool', 'default_pool', '--raw', '--subdir', '/home/airflow/.local/lib/python3.6/site-packages/airflow/example_dags/example_bash_operator.py', '--cfg-path', '/tmp/tmpg5lq4qgx', '--error-file', '/tmp/tmpghgim9b0']
[2021-02-23 14:57:10,311] {standard_task_runner.py:77} INFO - Job 23: Subtask runme_0
[2021-02-23 14:57:15,963] {logging_mixin.py:104} INFO - Running <TaskInstance: example_bash_operator.runme_0 2021-02-22T00:00:00+00:00 [running]> on host 0513e3a4eab6
[2021-02-23 14:57:18,535] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=example_bash_operator
AIRFLOW_CTX_TASK_ID=runme_0
AIRFLOW_CTX_EXECUTION_DATE=2021-02-22T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2021-02-22T00:00:00+00:00
[2021-02-23 14:57:18,544] {bash.py:135} INFO - Tmp dir root location: 
 /tmp
[2021-02-23 14:57:18,565] {bash.py:158} INFO - Running command: echo "example_bash_operator__runme_0__20210222" && sleep 1
[2021-02-23 14:57:18,796] {bash.py:169} INFO - Output:
[2021-02-23 14:57:18,912] {bash.py:173} INFO - example_bash_operator__runme_0__20210222
[2021-02-23 14:57:19,951] {bash.py:177} INFO - Command exited with return code 0
[2021-02-23 14:57:20,495] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=example_bash_operator, task_id=runme_0, execution_date=20210222T000000, start_date=20210223T145708, end_date=20210223T145720
[2021-02-23 14:57:20,793] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-02-23 14:57:20,914] {local_task_job.py:146} INFO - Task exited with return code 0
